src/styles/_catalog.scss

```
@@ -1,22 +1,29 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, 200px);
```

Тут можна було б не юзати repeat, він ж відпрацює тільки один раз
```suggestion
  grid-template-columns: 1, 200px;
```